File: Controller/src/Models/TrafficUpdate.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Models;

import java.util.List;

/**
 *
 * @author Niels
 */
public class TrafficUpdate
{
    public LightNumber LightId;
    public int Count;
    public List<Direction> DirectionRequests;
}

File: Simulator/Kruispunt/Assets/Scripts/TrafficUpdate.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.Serialization;

public class TrafficUpdate 
{
	public int LightID;
	public int Count;
	public int[] DirectionRequests;

	public TrafficUpdate(int lightID, int count, int[] directionRequests = null)
	{
		LightID = lightID;
		Count = count;
		DirectionRequests = directionRequests;
	}
}
